lib/batch_load/import/collecting_events/gpx_interpreter.rb

```diff
@@ -0,0 +1,96 @@
+module BatchLoad
+  class Import::CollectingEvents::GpxInterpreter < BatchLoad::Import
+
+    # SAVE_ORDER = [:georeference, :collecting_event]
+
+    def initialize(**args)
+      @collecting_events = {}
+      @ce_namespace = args.delete(:ce_namespace)
+      super(args)
+    end
+
+
+    # methode override for GPX processing which is quite different from CSV
+    # @return [Hash, nil]
+    def csv
+      gpx_file = GPX::GPXFile.new(gpx_file: @file.tempfile.path)
+      # @csv = Hash.from_xml(gpx.to_s)
```

Comment:
@TuckerJD Can we clear this up as well- and other comment blocks in this PR?  THanks.